go.mod

```diff
@@ -3,36 +3,37 @@ module github.com/cert-manager/sample-external-issuer
 go 1.20
 
 require (
-	github.com/cert-manager/cert-manager v1.12.0
-	github.com/go-logr/logr v1.2.4
-	github.com/stretchr/testify v1.8.3
-	k8s.io/api v0.27.2
-	k8s.io/apimachinery v0.27.2
-	k8s.io/client-go v0.27.2
-	k8s.io/utils v0.0.0-20230505201702-9f6742963106
-	sigs.k8s.io/controller-runtime v0.15.0
+	github.com/cert-manager/cert-manager v1.13.0-beta.0
+	github.com/cert-manager/issuer-lib v0.2.1-0.20230801145029-2a7fde6c4003
+	k8s.io/api v0.28.1
+	k8s.io/apimachinery v0.28.1
+	k8s.io/client-go v0.28.1
+	k8s.io/klog/v2 v2.100.1
+	sigs.k8s.io/controller-runtime v0.16.1
 )
 
+replace github.com/cert-manager/issuer-lib => github.com/inteon/issuer-lib v0.0.0-20230906123253-229987bd7c1e
```

Comment:
I believe this is necessary for integration but should be excluded from the PR into the sample repo

Reply:
This change was out-of-date, I updated the commit.